src/dmd/backend/evalu8.d

```diff
@@ -6,46 +6,51 @@
  *              Copyright (C) 2000-2018 by The D Language Foundation, All Rights Reserved
  * Authors:     $(LINK2 http://www.digitalmars.com, Walter Bright)
  * License:     $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost License 1.0)
- * Source:      $(LINK2 https://github.com/dlang/dmd/blob/master/src/dmd/backend/evalu8.c, backend/evalu8.c)
+ * Source:      $(LINK2 https://github.com/dlang/dmd/blob/master/src/dmd/backend/evalu8.d, backend/evalu8.d)
  */
 
-#if !SPP
-
-#include        <math.h>
-#include        <stdlib.h>
-#include        <stdio.h>
-#include        <string.h>
-
-#if _MSC_VER
-#define isnan _isnan
-#endif
-
-#include        "cc.h"
-#include        "oper.h"                /* OPxxxx definitions           */
-#include        "global.h"
-#include        "el.h"
-#include        "type.h"
+version (SPP)
```

Comment:
BTW: no module name here? (I won't object to module names without package if that's that's how you want to do it ;-) )

Reply:
No imports of it yet, I'll probably worry about that when imports happen. Right now, I just want to get it to work :-)